packages/go/openapi/src/schemas/model.oidc-provider.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+allOf:
+  - $ref: './model.components.int32.id.yaml'
+  - $ref: './model.components.timestamps.yaml'
+  - type: object
+    properties:
```

Comment:
Just gotta add the SSOProviderID since it will also be returned from the API
https://github.com/SpecterOps/BloodHound/blob/403c45c041f36c2e80db6c5922f298ee6484fff8/cmd/api/src/model/oidc_provider.go#L23